src/test/modules/test_misc/t/008_percona_server_version.pl

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/perl
+use strict;
+use warnings FATAL => 'all';
+use PostgreSQL::Test::Cluster;
+use PostgreSQL::Test::Utils;
+use Test::More;
+use lib 't';
+use Env;
+
+if (!defined($ENV{PERCONA_SERVER_VERSION}))
```

Comment:
I think if we have this test we should always run it.

Reply:
This check was just to keep checking the Percona version in GH and release workflows. In case some DEV is just using plain `make installcheck` or `make check ` on the command line, he should not be bothered about it; otherwise, if he is using ci_scripts bash files, it will kick in automatically.

Reply:
Why? Because on the face of it I disagree. I personally would want this error when I run it myself.

Reply:
Done.